plugins/versionpress/src/Utils/StringUtils.php

```diff
@@ -26,14 +26,14 @@ public static function verbToPastTense($verb)
     }
 
     /**
-     * Converts LF string to CRLF string
+     * Ensures LF line endings in a string
      *
-     * @param string $str LF line endings
-     * @return string CRLF line endings
+     * @param string $str LF or CRLF line endings
+     * @return string LF line endings
      */
-    public static function crlfize($str)
+    public static function ensureLf($str)
     {
-        return str_replace("\n", "\r\n", str_replace("\r\n", "\n", $str));
+        return str_replace("\\r\\n", "\\n", $str);
```

Comment:
> Ensures LF line endings in a string.

This is not true. Because of escaping it replaces string '\r\n' (`ASCII 0x005C 0x0072 0x005C 0x006E`) with '\n' (`ASCII 0x005C 0x006E`). Is it really what we want? I thought we want to replace CRLF (`ASCII 0x000D 0x000A`) with LF (`ASCII 0x000A`).